# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1436374510 14400
#      Wed Jul 08 12:55:10 2015 -0400
# Node ID 79d64c58daea1e58afd5e4a115a7eb9cb0580774
# Parent  820c5a4dd0abc5f2d1704c94e76dbb53993a1094
use $(abs_top_srcdir) in rules for running tests

* test/module.mk (check-local, check-jit): Use $(abs_top_srcdir)
instead of $(srcdir) for file and directory names to pass to the
run-octave script.

diff --git a/test/module.mk b/test/module.mk
--- a/test/module.mk
+++ b/test/module.mk
@@ -47,21 +47,21 @@ include test/fcn-handle-derived-resoluti
 include test/nest/module.mk
 
 all-local: test/.gdbinit
 
 test/.gdbinit: etc/gdbinit
 	@$(gdbinit_install_rule)
 
 check-local: 
-	cd test && ../run-octave $(RUN_OCTAVE_OPTIONS) --norc --silent --no-history $(srcdir)/test/fntests.m $(srcdir)/test
+	cd test && ../run-octave $(RUN_OCTAVE_OPTIONS) --norc --silent --no-history $(abs_top_srcdir)/test/fntests.m $(abs_top_srcdir)/test
 
 if AMCOND_HAVE_LLVM
 check-jit: test/sparse.tst test/bc-overload-tests.stamp test/$(octave_dirstamp)
-	cd test && ../run-octave $(RUN_OCTAVE_OPTIONS) --jit-compiler --norc --silent --no-history $(srcdir)/test/fntests.m $(srcdir)/test
+	cd test && ../run-octave $(RUN_OCTAVE_OPTIONS) --jit-compiler --norc --silent --no-history $(abs_top_srcdir)/test/fntests.m $(abs_top_srcdir)/test
 endif
 
 test/sparse.tst: test/build-sparse-tests.sh test/$(octave_dirstamp)
 	$(AM_V_GEN)rm -f $@-t $@ && \
 	$(srcdir)/test/build-sparse-tests.sh > $@-t && \
 	mv $@-t $@
 
 GENERATED_BC_OVERLOADS_DIRS := \
